src/main.rs

```diff
@@ -117,6 +123,69 @@ fn load_or_gen_keys(keys_file: &str) -> (PublicKey, SecretKey) {
     }
 }
 
+/// Modify settings
+fn sync_settings(cli_config: &mut CliConfig, settings: &mut Config) -> Result<(), Box<ConfigError>> {
+    if cli_config.udp_addr.is_none() {
+        cli_config.udp_addr = settings.get::<SocketAddr>("udp-address").ok();
+    }
+
+    if cli_config.tcp_addrs.len() == 0 {
+        cli_config.tcp_addrs = settings.get::<Vec<SocketAddr>>("tcp-address").unwrap_or(Vec::new());
+    }
+
+    if cli_config.keys_file.is_none() {
+        cli_config.keys_file = settings.get::<String>("keys-file").ok();
+    }
+
+    if cli_config.bootstrap_nodes.len() == 0 {
+        cli_config.bootstrap_nodes = settings.get::<Vec<BootstrapNode>>("bootstrap-nodes").unwrap_or(Vec::new());
+    }
+
+    if cli_config.threads_config.is_none() {
+        cli_config.threads_config = settings.get::<ThreadsConfig>("threads").ok();
+    }
+
+    if cli_config.log_type.is_none() {
+        cli_config.log_type = settings.get::<LogType>("log-type").ok();
+    }
+
+    if cli_config.motd == "This is tox-rs".to_owned() {
```

Comment:
Oh no. We should either use `--config` or other arguments. `--config` must be a unique argument.

Reply:
done